docs/manual/manual_telemetry.rst

```diff
@@ -0,0 +1,235 @@
+    Additional configuration may be necessary on the flight controller to properly intepret the received RPM data as RPM and not as ERPM. Refer to your flight controller's 
+    documentation.
+
+The telemetry sent by Vertiq modules when using :ref:`Hobby Protocols <manual_hobby>` is specified in the table below. The message
+consists of 10 bytes:
+
+	+----------+-------------------------+-----------------------+
+	| **Byte** | **Content**             | **Units**             |
+	+----------+-------------------------+-----------------------+
+	| 0        | Temperature             | Deg C                 |
+	+----------+-------------------------+-----------------------+
+	| 1        | Voltage (High Byte)     | cV                    |
+	+----------+-------------------------+-----------------------+
+	| 2        | Voltage (Low Byte)      |                       |
+	+----------+-------------------------+-----------------------+
+	| 3        | Current (High Byte)     | cA                    |
+	+----------+-------------------------+-----------------------+
+	| 4        | Current (Low Byte)      |                       |
+	+----------+-------------------------+-----------------------+
+	| 5        | Consumption (High Byte) | mAh                   |
+	+----------+-------------------------+-----------------------+
+	| 6        | Consumption (Low Byte)  |                       |
+	+----------+-------------------------+-----------------------+
+	| 7        | RPM (High Byte)         | RPM                   |
+	+----------+-------------------------+-----------------------+
+	| 8        | RPM (Low Byte)          |                       |
+	+----------+-------------------------+-----------------------+
+	| 9        | 8-bit CRC               |                       |
+	+----------+-------------------------+-----------------------+
+
+CRC Calculation
+----------------
+The CRC used with the telemetry message makes it possible to confirm that the received data is valid. The CRC is calculated on the payload of the
+telemetry message, which takes up the first 9 bytes of the message.
+
+The code blocks below show example code in C for calculating the CRC used in the telemetry message. This example 
+code is taken from the `KISS ESC telemetry standard <https://www.rcgroups.com/forums/showatt.php?attachmentid=8524039&d=1450424877>`_.
+
+.. code-block:: c
+
+    uint8_t update_crc8(uint8_t crc, uint8_t crc_seed){
+        uint8_t crc_u, i;
+        crc_u = crc;
+        crc_u ^= crc_seed;
+        for ( i=0; i<8; i++) crc_u = ( crc_u & 0x80 ) ? 0x7 ^ ( crc_u << 1 ) : ( crc_u << 1 );
+        return (crc_u);
+    }
+
+    uint8_t get_crc8(uint8_t *Buf, uint8_t BufLen){
+        uint8_t crc = 0, i;
+        for( i=0; i<BufLen; i++) crc = update_crc8(Buf[i], crc);
+        return (crc);
+    }
+
+Example Code for Extracting Telemetry Data

```

Comment:
Give people a very concrete example of how to parse the telemetry if they want to try it themselves.